### ISSUE TITLE
Accessibility: Site should have a skip navigation 

### ISSUE THREAD

**Issue by shawnthompson** [sara-sabr/vue-governance#14]
The site should have a [skip navigation](https://webaim.org/techniques/skipnav/).

I would suggest to make it as inclusive as possible and make it visible like the one found on [Law Office of Lainey Feingold](https://www.lflegal.com) site. 

WCAG Reference: [Understanding Success Criterion 2.4.1: Bypass Blocks](https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html)

**Comment by gcharest** [comment]
I have added a skip to main content but the example provided seems to have the link always visible?

**Comment by gcharest** [comment]
OK, so best practice would be to always keep visible then :thinking:

**Comment by gcharest** [comment]
(Because I do have a skipnav link, just shown when using tab on landing)

**Comment by gcharest** [comment]
![skipnav](https://user-images.githubusercontent.com/1690085/126578213-5f3fb5b9-a48b-43cc-bac4-1790f6c424c2.gif)
